src/tagging/reducers/reducers.js

```diff
@@ -1,5 +1,6 @@
 import * as actionsConstants from '../actions/actions';
-// state = state.assignedTags
+
+/* ================== SECTION: state = state.assignedTags ================== */
```

Comment:
Can we rather use JS DOC notation instead of these comments?

Reply:
Yes, but I didn't found how to make a section of code.